portfolio/src/main/webapp/script.js

```diff
@@ -19,11 +19,17 @@ async function getComments() {
   const maxComments = document.getElementById('max-comments').value;
   const response = await fetch('/list-comments?max-comments=' + maxComments);
   const comments = await response.json();
-  const commentsElement = document.getElementById('comments-container');
+  const commentsElement = document.getElementById('comments-list');
   commentsElement.innerHTML = '';
 
-  for (const comment of comments) {
-    commentsElement.appendChild(createCommentElement(comment))
+  for (var comment of comments) {
+    commentsElement.appendChild(createCommentElement(comment));
+  }
+
+  if (commentsElement.innerHTML === '') {
+    document.getElementById("comment-section").style.visibility="hidden";
```

Comment:
nit: can you change " to ' and also add spaces around the `=`?

Reply:
Will do!